src/apps/integrations/loris/domain/loris_integrations.py

```diff
@@ -0,0 +1,29 @@
+import uuid
+
+from apps.integrations.db.schemas import IntegrationsSchema
+from apps.integrations.domain import AvailableIntegrations
+from apps.shared.domain import InternalModel
+
+
+class IntegrationMeta(InternalModel):
+    hostname: str
+    username: str
+    password: str
+    project: str | None
+
+
+class Integrations(InternalModel):
+    applet_id: uuid.UUID
+    type: AvailableIntegrations
+    configuration: IntegrationMeta
+
+    @classmethod
+    def from_schema(cls, schema: IntegrationsSchema):
+        return cls(applet_id=schema.applet_id, type=schema.type, configuration=schema.configuration)
+
+
+class IntegrationsCreate(InternalModel):
```

Comment:
If you add `type: AvailableIntegrations` to `IntegrationsCreate` we can get route level validation for free, though it will slightly alter the route to remove `/type` from the path.

This feels desirable as this endpoint (current ticket) should replace `enable_integration`